allennlp/models/semantic_parsing/wikitables/wikitables_erm_semantic_parser.py

```diff
@@ -53,6 +57,10 @@ class WikiTablesErmSemanticParser(WikiTablesSemanticParser):
         Should we normalize the log-probabilities by length before renormalizing the beam? This was
         shown to work better for NML by Edunov et al., but that many not be the case for semantic
         parsing.
+    checklist_cost_weight : ``float``, optional (default=0.8)
```

Comment:
This seems high for a model that's initialized with MML...

Reply:
Yes, I found that 0.6 is a good value for nlvr. Changed the defaults.